tests/vitest/composition/BarChart.test.ts

```
@@ -11,9 +11,7 @@ vi.mock(`$lib/composition/parse`, () => ({
     )
     const percentages: Record<string, number> = {}
     for (const [element, amount] of Object.entries(comp)) {
-      if (typeof amount === `number`) {
-        percentages[element] = (amount / total) * 100
-      }
+      if (typeof amount === `number`) percentages[element] = (amount / total) * 100
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)